testing/integration/postman/BioHubBC-Environment-DEV.postman_environment.json

```diff
@@ -1,5 +1,5 @@
 {
-	"id": "65ce4931-9611-4df9-bca1-bc2340e0810a",
+	"id": "d9ec52f3-e88a-45ee-867d-116e25294e38",
```

Comment:
We can just revert this file entirely, as we never use the object store env vars anywhere in postman.